### ISSUE TITLE
Relative music strings

### ISSUE THREAD

**Issue by fnord-repeater** [leonawicz/tabr#36]
Hello, 

Thanks for implementing string numbers in music strings. I have another request.

Could you add an attribute for lilypond's `\relative` note entry mode?

I feel this would make transcription easier.

Thanks

**Comment by leonawicz** [comment]
Several months back prior to the last big release I put a lot of thought into this. I decided not to pursue it for various reasons including

1. Low priority for tabr, even if it's a popular approach among actual LilyPond users.

2. Potential technical difficulty of integrating nicely throughout the package (possibly avoidable, see below).

3. If I did implement something, I would not follow the very nuanced and esoteric LilyPond rule set for [relative octave entry](http://lilypond.org/doc/v2.18/Documentation/notation/writing-pitches). For any use case where this level of one to one match with LilyPond itself is important, I recommend just using LilyPond directly.

I would do something much simpler and consistent with the way in which `as_music()` allows duration and string number to simply repeat, nothing more. Even so it would still require an additional argument to toggle on relative octave numbering (though I guess I wouldn't call it "relative"). This is important because otherwise any unspecified octave is meant to be octave 3.

As for making this more technically feasible, it would only be an option for data entry in `as_music()` but then `as_music()` would immediately fill in the implicit octave numbers explicitly, internally reverting everything to the usual format. Then the rest of the package would not be impacted. I think this is probably what you are suggesting. I'm open to it, but considering the simpler, tabr-consistent (not LilyPond rules) way in which I'd do it, is this even still useful? I would not expect much use of the feature in general.
